analyzer/src/funTest/kotlin/managers/NpmFunTest.kt

```diff
@@ -141,15 +140,30 @@ class NpmFunTest : WordSpec({
                 yamlMapper.readValue<ProjectAnalyzerResult>(it)
             }
 
-            val actualResult = createNpm().resolveSingleProject(definitionFile, resolveScopes = true)
+            val actualResult = resolveSingleProject(definitionFile, resolveScopes = true)
 
             actualResult.withInvariantIssues() shouldBe expectedResult.withInvariantIssues()
         }
     }
 })
 
-private fun createNpm(
-    analyzerConfig: AnalyzerConfiguration = AnalyzerConfiguration(),
-    repoConfig: RepositoryConfiguration = RepositoryConfiguration()
-) =
-    Npm("NPM", USER_DIR, analyzerConfig, repoConfig)
+private fun resolveSingleProject(
```

Comment:
Maybe call this `resolveNpmSingleProject` to distinguish this more from `npm.resolveSingleProject`.